File: CS/GridControlCRUD.Common/Northwind/Product.cs
using System;
using System.Linq;

namespace DevExpress.CRUD.Northwind {
    public class Product {
        public long Id { get; set; }
        public string Name { get; set; }
        public long CategoryId { get; set; }
        public virtual Category Category { get; set; }
        public string QuantityPerUnit { get; set; }
        public decimal? UnitPrice { get; set; }
        public short? UnitsInStock { get; set; }
        public short? UnitsOnOrder { get; set; }
        public short? ReorderLevel { get; set; }
        public bool Discontinued { get; set; }
        public string EAN13 { get; set; }
    }
}

File: VB/GridControlCRUD.Common.Northwind/Northwind - VB/Product.vb
Imports System
Imports System.Linq

Namespace DevExpress.CRUD.Northwind
	Public Class Product
		Public Property Id() As Long
		Public Property Name() As String
		Public Property CategoryId() As Long
		Public Overridable Property Category() As Category
		Public Property QuantityPerUnit() As String
		Public Property UnitPrice() As Decimal?
		Public Property UnitsInStock() As Short?
		Public Property UnitsOnOrder() As Short?
		Public Property ReorderLevel() As Short?
		Public Property Discontinued() As Boolean
		Public Property EAN13() As String
	End Class
End Namespace
